nextflow_schema.json

```diff
@@ -604,11 +579,6 @@
                     "description": "Specifies whether or not to skip calling of repeat expansions.",
                     "fa_icon": "fas fa-toggle-on"
                 },
-                "skip_smncopynumbercaller": {
-                    "type": "boolean",
-                    "description": "Specifies whether or not to skip smncopynumbercaller.",
-                    "fa_icon": "fas fa-toggle-on"
-                },
                 "skip_snv_annotation": {
```

Comment:
might make sense to add more general skip option here? like skip "annotation"? I think it would be a rare case to have lets say snvannotation on while svannotaton off?

Reply:
Depends on the use case, I believe. There are scenarios where someone might have updated databases for SNVs, and in such cases, they might not want to run SV annotation.